contributing_to_docs/doc_guidelines.adoc

```diff
@@ -122,6 +114,66 @@ To use a discrete heading, just add `[discrete]` to the line before your unique
 == Managing authorization policies
 ----
 
+== Anchoring titles and section headings
+
+All titles and section headings must have a anchor. The anchor ID must be similar to the title or section heading.
+
+*Example anchor in an assembly file*
+----
+[id="understanding-the-monitoring-stack"]
+= Understanding the monitoring stack
+----
+
+[NOTE]
+====
+Do not include line spaces between the anchor and the section title.
+====
+
+You must add the `{context}` variable to the end of anchor IDs in module files. When called, the `{context}` variable is resolved into the value declared in the `:context:` attribute in the corresponding assembly file. This enables cross-referencing to module IDs in context to a specific assembly and is useful when a module is included in multiple assemblies.
+
+*Example anchor in a module file*
```

Comment:
It looks like these bold sections are directly above the example syntax, but there is direction above it for module anchors. I would make each of these bold sections the heading for the respective type:

* Assembly
* Module
* Prerequisites/Additional resources/Next steps

Or make them into == headings like how you have it for "Creating unique anchor IDs for 'Prerequisites', 'Additional resources', and 'Next steps' titles"

And then probably use something other than bold for labeling the example syntax. The bold just makes it look too prominent, like it's its own section heading.

Reply:
Thank you for highlighting this. I have now created subsections at the same heading level for each of the three topics in that list and have introduced the examples in plain text.